target/data/publication.js

```diff
@@ -559,7 +559,7 @@ var publicationJsonData={
       "tpk": null,
       "max_rpm": null,
       "images": [],
```

Comment:
doesn't the below change break the unit tests @rnllv ?

Reply:
Not the unit tests. It was added for our parser tests. Nope, it's not breaking since I was just searching for the text within the two percentage symbols.